### PR TITLE
Disable sending blank text field when textarea becomes enabled

### LINKED ISSUE #772: Remove simple survey "feedback" link from footer
image

Beside text-only, in the links along the bottom left of the footer.

No longer necessary now that we have the in-site survey. Double check that there aren't other links to the old external feedback survey kicking around (on the about route maybe?).

### PR DESCRIPTION
**Pull request by GarvinH**
closes #772 

image

VS

image

### CODE REVIEW

**Comment by GarvinH** [comment]
A11y now looks like this if this is okay

image

File: client/src/InfoLab/InfoLab.js [hunk 1]
```diff
@@ -28,6 +30,13 @@ export default class InfoLab extends React.Component {
         <TM k="lab_title" el="h1" />
         <div className="medium_panel_text">
           <TM k="lab_intro_text" />
+          <button
```
Comment (ktw1016) [inline client/src/InfoLab/InfoLab.js:33]: Maybe we should disable the survey button in PageDetails on InfoLab page
(I did the same for about route #766 )

File: client/src/components/EmailFrontend.js [hunk 2]
```diff
@@ -167,8 +167,27 @@ class EmailFrontend extends React.Component {
       )
       .value();
 
+    const all_connected_user_fields_are_filled = _.chain(user_fields)
+      .toPairs()
+      .filter(([field_name, field_val]) =>
+        _.includes(_.keys(field_val), "connection")
```
Comment (ktw1016) [inline client/src/components/EmailFrontend.js:173]: I'm guessing here but is it necessary to put two filters here?
```
      .filter(
        ([field_name, field_val]) =>
          _.includes(_.keys(field_val), "connection") &&
          _.includes(_.keys(completed_template), field_val.connection.name) &&
          _.includes(
            completed_template[field_val.connection.name],
            field_val.connection.enable
          )
      )
```
You should check if that works !

**Review by ktw1016 — APPROVED** [review]
2 super minor comments, other than good to go

File: client/src/InfoLab/InfoLab.js (hunk 1 above)
Comment (GarvinH) [reply client/src/InfoLab/InfoLab.js:36]: 👍 I changed it back so that it still shows the report a problem and github version banners

**Pull request merged**